src/util/misc.rs

```diff
@@ -34,31 +80,46 @@ pub fn remove_expired(pastas: &mut Vec<Pasta>, args: &Args) {
             // keep
             true
         } else {
+            log::debug!("Removing expired pasta ID: {} ({})", p.id, p.id_as_animals(&args.hash_ids));
```

Comment:
lets extract this whole log block to a function named: `log_removal_details` (or another name)